Formula/lammps.rb

```diff
@@ -0,0 +1,49 @@
+class Lammps < Formula
+  desc "Molecular Dynamics Simulator"
+  homepage "http://lammps.sandia.gov"
+  url "http://lammps.sandia.gov/tars/lammps-11Aug17.tar.gz"
+  # lammps releases are named after their release date. We transform it to
+  # YYYY.MM.DD (year.month.day) so that we get a comparable version numbering (for brew outdated).
+  # We only track "stable" releases as announced on the LAMMPS homepage.
+  version "2017.08.11"
+  sha256 "33431329fc735fb12d22ed33399235ef9506ba759a281a24028de538822af104"
+  revision 2
+
+  depends_on "fftw"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on :fortran
+  depends_on :mpi => [:cxx, :f90, :recommended]
+
+  def install
+    %w[serial mpi].each do |variant|
+      cd "src" do
+        system "make", "clean-all"
+        system "make", "yes-standard"
+
+        # Disable some packages for which we do not have dependencies, that are
+        # deprecated or require too much configuration.
+        %w[gpu kim kokkos mscg meam mpiio poems reax voronoi].each do |package|
+          system "make", "no-#{package}"
+        end
+
+        system "make", variant,
+                       "LMP_INC=-DLAMMPS_GZIP",
+                       "FFT_INC=-DFFT_FFTW3 -I#{Formula["fftw"].opt_prefix}/include",
+                       "FFT_PATH=-L#{Formula["fftw"].opt_prefix}/lib",
+                       "FFT_LIB=-lfftw3",
+                       "JPG_INC=-DLAMMPS_JPEG -I#{Formula["jpeg"].opt_prefix}/include -DLAMMPS_PNG -I#{Formula["libpng"].opt_prefix}/include",
+                       "JPG_PATH=-L#{Formula["jpeg"].opt_prefix}/lib -L#{Formula["libpng"].opt_prefix}/lib",
+                       "JPG_LIB=-ljpeg -lpng"
+
+        bin.install "lmp_#{variant}"
```

Comment:
have you requested a make install target

Reply:
They're moving to cmake (❤️). I'll make a request anyway, but in a few months this should all be gone!